packages/pluggableWidgets/line-chart-native/src/components/LineChart.tsx

```diff
@@ -160,7 +196,12 @@ export function LineChart(props: LineChartProps): ReactElement | null {
                                     <VictoryChart
                                         height={chartDimensions?.height}
                                         width={chartDimensions?.width}
-                                        padding={style.grid?.padding}
+                                        padding={{
+                                            top: style.grid?.paddingTop,
```

Comment:
We might have a problem here if the user defined paddingVertical, paddingHorizontal or just padding. maybe we need to map them

Reply:
Changed it :)